### ISSUE TITLE
Generated Map Setter tests incorrect variable for null

### ISSUE THREAD

**Issue by nebhale** [immutables/immutables#777]
In 2.6.0, an example generated map setter looks like this:

```java
public final Builder environmentJsons(@Nullable Map<String, ? extends Object> entries) {
  if (environmentJsons == null) {
    this.environmentJsons = null;
    return this;
  }
  this.environmentJsons = new LinkedHashMap<String, Object>();
  return putAllEnvironmentJsons(entries);
}
```

Given this code, it does not appear to be possible to use this method to set a non-`null` value with `entries`.  I believe that the very first line of the method should be `if (entries == null) {`.

**Comment by elucash** [comment]
definitely a problem, thank you for reporting

**Comment by elucash** [comment]
released in 2.6.1, thank you!

**Comment by nebhale** [comment]
Fix confirmed.  Thanks for the quick turn around.
